projects/plugins/jetpack/modules/search/instant-search/lib/filters.js

```diff
@@ -46,6 +46,37 @@ export function getFilterKeys(
 	return [ ...keys ];
 }
 
+/**
+* Get a list of provided static filters.
+*
+* @return []Object
+*/
+export function getAvailableStaticFilters () {
+	if ( ! window[ SERVER_OBJECT_NAME ] ) {
+		return [];
+	}
+
+	if ( ! window[ SERVER_OBJECT_NAME ].staticFilters ) {
+		return [];
+	}
```

Comment:
(Nit) Could simplify to the following:

```suggestion
	if ( ! window[ SERVER_OBJECT_NAME ]?.staticFilters ) {
		return [];
	}
```

Reply:
Good point